firebase-perf/src/main/java/com/google/firebase/perf/metrics/NetworkRequestMetricBuilder.java

```diff
@@ -293,7 +293,7 @@ public NetworkRequestMetricBuilder setResponseContentType(@Nullable String conte
     if (isValidContentType(contentType)) {
       builder.setResponseContentType(contentType);
     } else {
-      logger.info("The content type of the response is not a valid content-type:" + contentType);
+      logger.warn("The content type of the response is not a valid content-type:" + contentType);
```

Comment:
Should this be a warning?

Reply:
Refer to my previous comment:
"My original thought is that this one can be either Debug or Warn because contentType is extracted from response. We check contentType length and disallow control characters but this could be a false alarm.
I updated to warn to match that this contentType is not valid or expected from Fireperf's standard."